### ISSUE TITLE
Name reused expressions

### ISSUE THREAD

**Issue by oliverjam** [mhtien/js-calculator#5]
If you're going to use a complex-ish expression multiple times it's a good idea to give it a descriptive name.

https://github.com/mhtien/js-calculator/blob/d220d0209c55d1f10150d0f5dc68ab42bffe1d78/script.js#L218-L235

`operatorsArray[i]` isn't too hard to parse, but you could save yourself some typing/reading later on with `const operator = operatorsArray[i]` at the top.

**Comment by mhtien** [comment]
Thanks again for the tips! It does read a lot clearer now :+1: 

[View changes here](https://github.com/mhtien/js-calculator/commit/3b6f15106481586a7380295ab6b52f8f884a890b)
